frontend/src/components/molecules/AddTransaction.tsx

```diff
@@ -1,15 +1,67 @@
 import moment from 'moment';
-import React from 'react';
+import React, { useReducer, ReactElement } from 'react';
 import styled from 'styled-components';
 import { useAuthState } from '../../store/contexts/auth';
 import { useTransactionDispatch } from '../../store/contexts/transactions';
 import { TransactionActions } from '../../store/reducers/transactions';
 import Collapsable from '../atoms/Collapsable';
 import Form from './Form';
+import SnackBarContainer from '../atoms/SnackBarContainer';
+
+const initialState = { snax: <div></div>, content: '' };
+
+type stateType = {
```

Comment:
All types should be Capitalized.

Reply:
I recommend setting up tslint and prettier in your workspace. If not, run `tslint --fix -p .` in the root of the `frontend` folder.